specify/specify7
# Prevent from saving invalid latitude and longitude

Fixes #4822

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Go to Data Entry
- Create new Locality
- Enter a value outside -90 and 90 in the Latitude field
- Enter a value outside -180 and 180 in the Longitude field
- [ ] Verify that Save is blocked and a tooltip displays the reason on the field
